cmd/api/src/daemons/datapipe/agi.go

```diff
@@ -160,11 +160,13 @@ func TagActiveDirectoryTierZero(ctx context.Context, db database.Database, graph
 
 	if autoTagT0ParentObjectsFlag, err := db.GetFlagByKey(ctx, appcfg.FeatureAutoTagT0ParentObjects); err != nil {
 		return err
+	} else if autoTagMembersFlag, err := db.GetFlagByKey(ctx, appcfg.FeatureAutoTagT0ADMembers); err != nil {
```

Comment:
I tried to test this locally and was unable to build the app. There's a change in main that breaks this line. The `db` variable is no longer present in the latest code. It looks like it was changed in #816.

Reply:
Fixed